.github/workflows/osrm-backend.yml

```diff
@@ -34,12 +36,38 @@ jobs:
         node-version: 16
     - run: node --version
     - run: npm --version
+    - name: Prepare environment
+      shell: bash
+      run: |
+        PACKAGE_JSON_VERSION=$(node -e "console.log(require('./package.json').version)")
+        echo PUBLISH=$([[ "${GITHUB_REF:-}" == "refs/tags/v${PACKAGE_JSON_VERSION}" ]] && echo "On" || echo "Off") >> $GITHUB_ENV
     - run: npm install --ignore-scripts
     - run: npm link --ignore-scripts
     - uses: microsoft/setup-msbuild@v1.1
     - name: Build
       run: |
         .\scripts\ci\windows-build.bat
+    - name: Run node tests
+      shell: bash
+      run: |
+        ./lib/binding/osrm-datastore.exe test/data/ch/monaco.osrm
+        node test/nodejs/index.js | faucet
+    - name: Build Node package
+      if: ${{ env.PUBLISH == 'On' }}
+      run: bash ./scripts/ci/node_package.sh
+    - name: Publish Node package
+      if: ${{ env.PUBLISH == 'On' }}
+      uses: ncipollo/release-action@v1
```

Comment:
Couldn't it check as well, but I expect that it should work as on other Linux/macOS jobs.

Reply:
Hmm, well the Windows build was published, which is great.
https://github.com/mjjbell/osrm-backend/releases/tag/v9.90.22

But the Linux packaging failed.
https://github.com/mjjbell/osrm-backend/actions/runs/3063972998/jobs/4946640880

Not sure if it's related to these changes, looks like it might be a Conan issue.

Reply:
I think I fixed it in https://github.com/Project-OSRM/osrm-backend/pull/6360 (via RPATH changes, tbh I am not sure that it is caused by Conan, but may be)

Reply:
Yep, looks like that fixes it: https://github.com/mjjbell/osrm-backend/actions/runs/3073294832